### ISSUE TITLE
WALLET ACCESS 

### ISSUE THREAD

**Issue by mike1968mi** [bitpay/copay#6089]
So I do need an internet connection to recover the wallet with the phrase so it IS stored on Copay servers so my Bitcoin must also be there as well.

All I want is MY Bitcoin funds, that's all. I'm not asking for anything that does not belong to me.

By due process of law, my wallet funds must be returned to me from Copay. You must give me access to the service you provided me with and use another form of verification.

 **Copay breached their terms and services by using false statements about the service and misleading clients.**

At the time of my deposit in to my Copay wallet Bitcoin was at $1,800 and is now over $2,000 and you must pay the amount I have lost on my investment as well.

**Comment by gabrielbazan7** [comment]
It has been answered here #6087

**Comment by dabura667** [comment]
No.

Just admit that your own inability to read the terms of service and the multiple warning windows that pop up lead to your lack of backup, which lead to your eventual loss.

I understand that human beings will believe anything to remove blame from themselves, but you have been told on multiple occasions exactly how Copay works, and you keep coming to conclusions that are the exact opposite of what you have been told.

Do you want proof that the phrase can generate the bitcoin addresses and private keys offline?

1. Go to https://bip32jp.github.io/english/
2. Create a new wallet on Copay, and type the 12 word phrase on a notepad document etc. on the same computer. Also copy down the address shown on the Receive tab for the new wallet (make sure the name of the new wallet is shown underneath the address)
3. Turn off the internet, unplug any LAN cables etc. with your browser still open to the page I linked.
4. Paste your 12 word phrase over the existing 12 word phrase.
5. Click the Derivation Path and change to BIP 44 ... (receiving) ...
6. Notice the private key is shown and the address matches the first address of the new wallet you made.
7. This proves that the website's javascript could generate the address and corresponding private key without internet.

Just because you can't understand something doesn't make it not true.

**Comment by mike1968mi** [comment]
Another incredibly rude response. The worst so called "customer service" ever.

I still have not received an answer about recovering the wallet with no internet access.

**Comment by bitjson** [comment]
Thanks for the thoughtful comment @dabura667. (And for your continued contributions!)

**Comment by dabura667** [comment]
@mike1968mi

This is an open source project. I am just some random guy on the internet that knows how Copay works because I have volunteered my time to work on the app a little bit in the past.

I don't represent Bitpay or Copay, and I don't owe you anything.

I have offered to teach you more about how bitcoin works and your response has always been rude, curt, and completely oblivious of every single thing I have told you. You never even accepted my offer to teach you how it works.

I understand why you don't want to know how it works. Because if you knew how it works you would need to blame yourself for your loss, and you don't want to accept that fact and therefore are screaming the same thing over and over and over.

The longer you wait to take your phone into a data recovery store, the more likely your phrase may have been corrupted or disappeared from the broken device's memory.

If I were you, I would be desparately trying to recover the data on my broken phone.
